src/plugins/intel_cpu/tests/functional/subgraph_tests/src/denormals_on_movement_instruction_check.cpp

```diff
@@ -0,0 +1,123 @@
+        std::tie(inputShapes, eltwiseOpType, targetName) = obj.param;
+        std::ostringstream results;
+
+        results << "IS=(";
+        for (const auto& shape : inputShapes) {
+            results << ov::test::utils::partialShape2str({shape.first}) << "_";
+        }
+        results << ")_TS=(";
+        for (const auto& shape : inputShapes) {
+            for (const auto& item : shape.second) {
+                results << ov::test::utils::vec2str(item) << "_";
+            }
+        }
+        results << "eltwiseOpTypes=" << eltwiseOpType << "_";
+        results << "targetDevice=" << targetName;
+
+        return results.str();
+    }
+
+    void generate_inputs(const std::vector<ov::Shape>& targetInputStaticShapes) override {
+        constexpr unsigned seed = 1u;
+        constexpr uint32_t denormalsRange = (0xffffffffu >> 9u) - 1;
+        testing::internal::Random random(seed);
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (size_t i = 0; i < funcInputs.size(); ++i) {
+            const auto& funcInput = funcInputs[i];
+            auto tensor = ov::Tensor{funcInput.get_element_type(), targetInputStaticShapes[i]};
+            auto data_ptr_int32 = static_cast<int32_t*>(tensor.data());
+            size_t shape_size = ov::shape_size(targetInputStaticShapes[i]);
+            for (size_t j = 0; j < shape_size; ++j) {
+                auto denormal = random.Generate(denormalsRange) + 1;
+                int32_t tmp;
+                memcpy(&tmp, &denormal, sizeof(int32_t));
+                data_ptr_int32[j] = tmp;
+            }
+            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
+        }
+    }
+
+protected:
+void SetUp() override {
+    // force denormals flag is on
+    configuration.insert({"CPU_DENORMALS_OPTIMIZATION", InferenceEngine::PluginConfigParams::YES});
+    // force eltwise node path
+    configuration.insert({InferenceEngine::PluginConfigInternalParams::KEY_SNIPPETS_MODE,
+                              InferenceEngine::PluginConfigInternalParams::DISABLE});
+    std::vector<InputShape> inputShapes;
+    EltwiseTypes eltwiseOpType;
+    std::tie(inputShapes, eltwiseOpType, targetDevice) = this->GetParam();
+
+    init_input_shapes(inputShapes);
+
+    ov::ParameterVector params;
+    for (auto&& shape : inputDynamicShapes)
+        params.push_back(std::make_shared<ov::op::v0::Parameter>(ElementType::i32, shape));
+    auto eltwise = ngraph::builder::makeEltwise(params[0], params[1], eltwiseOpType);
+    function = std::make_shared<ngraph::Function>(eltwise, params, "FTZ/DAZ_impact_on_data_movement_instructions");
+}
+};
+
+TEST_P(DenormalOnDataMovementInstructionCheck, CompareWithRefs) {
+    run();
+}
+
+namespace {
+
+std::vector<std::vector<ngraph::Shape>> inputShapes = {
+    {{1, 1, 1, 1}, {1, 1, 1, 1}},     // test uni_vmovss with load_scalar
+    {{1, 3, 2, 32}, {1, 3, 2, 32}},   // test uni_vmovups with load_vector
```

Comment:
Here, the "uni_vmovups" and "uni_vmovss" will be used in the eltwise ?
And, do we need to limit this test to "defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)"?